sky/skylet/providers/lambda_labs/node_provider.py

```diff
@@ -0,0 +1,198 @@
+import logging
```

Comment:
nit: It would be nice to have type annotations.

Reply:
Done.